ts/Core/Series/SeriesDefaults.ts

```diff
@@ -552,6 +552,20 @@ const seriesDefaults: PlotOptionsOf<Series> = {
      * @apioption plotOptions.series.negativeColor
      */
 
+    /**
+     * Whether or not data-points with the value of `null` should be interactive.
+     * When this is set to `true`, tooltips may highlight these points, and this
+     * option also enables keyboard navigation for such points. Format options
+     * for such points include [`nullFormat`](#tooltip.nullFormat) and [`nullFormater`](#tooltip.nullFormatter)
+     *
+     * @sample {highcharts} highcharts/series/null-interaction/
+     *         Chart with interactive `null` points
+     *
+     * @type      {boolean|undefined}
+     * @product   highcharts highstock
+     * @apioption plotOptions.line.nullInteraction
```

Comment:
This should be `@apioption plotOptions.series.nullInteraction`, because we can use it for all series? And should be excluded for series like pie.